sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/AppConfigurationBootstrapConfiguration.java

```diff
@@ -130,8 +110,9 @@ public AppConfigurationPropertySourceLocator sourceLocator(AppConfigurationPrope
      * @return ClientStore
      */
     @Bean
-    public ClientStore buildClientStores(AppConfigurationProperties properties,
-        AppConfigurationProviderProperties appProperties, ConnectionPool pool, Environment env,
+    @ConditionalOnMissingBean
+    AppConfigurationReplicaClientBuilder replicaClientBuilder(AppConfigurationProperties properties,
```

Comment:
Can the `AppConfigurationReplicaClientBuilder` implements the `EnvironmentAware`?